hummingbot/connector/exchange/coinbase_advanced_trade/coinbase_advanced_trade_exchange.py

```diff
@@ -1025,10 +1025,9 @@ async def _all_trade_updates_for_order(self, order: InFlightOrder) -> List[Trade
         trade_updates = []
         if order.exchange_order_id is not None:
             order_id: str = order.exchange_order_id
-            product_id: str = await self.exchange_symbol_associated_to_pair(trading_pair=order.trading_pair)
+            # product_id: str = await self.exchange_symbol_associated_to_pair(trading_pair=order.trading_pair)
             params = {
-                "product_id": product_id,
-                "order_id": order_id
+                "order_ids": order_id
```

Comment:
Should this be [order_id]?

Reply:
correct @MementoRC

Reply:
the api noted string thou @MementoRC and not list format